README.md

```diff
@@ -1,94 +1,27 @@
 # SIG-Release
 
-Welcome to SIG-Release, home of release and feature management.
-If you are new to this repository, you might find the [getting started guide](docs/getting-started.md) useful.
-If you are interested in the current state of planning, have a look at our [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
+## Intro
 
-## The planning and roadmap process
+Welcome to SIG-Release, home of release and feature management. If you are new to this repository, you might find the getting started guide useful. If you are interested in the current state of planning, have a look at our Release Planning Board.
 
-We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
+[Catena-X Standards](https://catena-x.net/de/standard-library) are the foundation for certifying any software product operating in the Catena-X data space. Those standards are the binding reference to obtain a valid certificate.
 
-While every repository in the [eclipse-tractusx](https://github.com/eclipse-tractusx) GitHub organization
-has its own issue management, the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
-is used to align the overarching Tractus-X releases.
+To make adopting the latest [Catena-X Standards](https://catena-x.net/de/standard-library) easier, reference implementations are provided through the Eclipse Tractus-X Project
 
-### How can I get involved
+## Release Cycle - major vs. minor release
 
-In case you experienced a bug, unexpected behaviour, or you want to propose enhancements to Eclipse Tractus-X,
-feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose) and describe your request.
-Please be aware, that not every feature request can be integrated and that we also cannot treat every issue with the highest priority.
+Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release tractus-x-release.md following [CalVer](https://calver.org) is released.
```

Comment:
Tractus-X operates on a quarterly release cycle. Each quarter, a new [tractus-x release] (tractus-x-release.md) following [CalVer](https://calver.org) is released.

Reply:
I don't think thats correct. Tractus-X is a name and we are referenzing a document with the same name. I fixed the link

Reply:
pls update to:
Tractus-X operates on a quarterly release cycle. Each quarter, a new [Tractus-X release] (tractus-x-release.md) following [CalVer](https://calver.org/) is released.
=> mind the necessary URL link in normal brackets

Reply:
done